From b0b740f38da6d4584417d65a7c97fd4faf982cf6 Mon Sep 17 00:00:00 2001
From: Gary Lin <glin@suse.com>
Date: Wed, 4 Dec 2019 10:28:28 +0800
Subject: [PATCH] pesigcheck: remove superfluous type settings

When setting the type of reason in check_signature(), the type was
accidentally set as "siBuffer". Since the type is already set as
"SIGNATURE", we only need to remove those two lines of code.

Fixes: https://github.com/rhboot/pesign/issues/55

Signed-off-by: Gary Lin <glin@suse.com>
---
 src/pesigcheck.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/pesigcheck.c b/src/pesigcheck.c
index 524cce3..d197c30 100644
--- a/src/pesigcheck.c
+++ b/src/pesigcheck.c
@@ -318,7 +318,6 @@ check_signature(pesigcheck_context *ctx, int *nreasons,
 			reason->type = SIGNATURE;
 			reason->sig.data = data;
 			reason->sig.len = datalen;
-			reason->type = siBuffer;
 			nreason += 1;
 			is_invalid = true;
 		}
@@ -330,7 +329,6 @@ check_signature(pesigcheck_context *ctx, int *nreasons,
 			reason->type = SIGNATURE;
 			reason->sig.data = data;
 			reason->sig.len = datalen;
-			reason->type = siBuffer;
 			nreason += 1;
 			has_valid_cert = true;
 		}
-- 
2.24.0

